strings/brain_damage_lines.json

```diff
@@ -61,7 +61,8 @@
         "@pick(semicolon)N-NYAAAAAA~",
         "@pick(bug)",
         "@pick(semicolon)wtf??????????? @pick(bug)",
-        "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)"
+        "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
+		"@pick(semicolon)okay i'm reporting shitcurity don't grab prisoners with you is 1 shitcurity is 2 and asscurity is 3 = permaban i'm reporting"
```

Comment:
what was our stance on OOC in IC in code again?
I believe it was NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO.
(the 'permaban' bit)